Source/Pages/Gallery/YPLibraryVC+CollectionView.swift

```diff
@@ -144,9 +144,18 @@ extension YPLibraryVC: UICollectionViewDelegate {
         let isVideo = (asset.mediaType == .video)
         cell.durationLabel.isHidden = !isVideo
         cell.durationLabel.text = isVideo ? YPHelper.formattedStrigFrom(asset.duration) : ""
-        cell.multipleSelectionIndicator.isHidden = !isMultipleSelectionEnabled || (isMultipleSelectionEnabled && isVideo)
+
+        if !YPImagePickerConfiguration.shared.library.allowPhotoAndVideoSelection {
+            cell.multipleSelectionIndicator.isHidden = !isMultipleSelectionEnabled || (isMultipleSelectionEnabled && isVideo)
+            cell.isUserInteractionEnabled = !(isMultipleSelectionEnabled && isVideo)
+        } else {
+            cell.multipleSelectionIndicator.isHidden = !isMultipleSelectionEnabled
+            cell.isUserInteractionEnabled = true
+        }
+
+
         cell.isSelected = !disableAutomaticCellSelection && currentlySelectedIndex == indexPath.row && selectedItems.contains(where: { $0.assetIdentifier == asset.localIdentifier })
-        cell.isUserInteractionEnabled = !(isMultipleSelectionEnabled && isVideo)
+        cell.isUserInteractionEnabled = true
```

Comment:
Should this line be removed since it is already set above

Reply:
+1